### ISSUE TITLE
"installing standalone, skipping download" shown for non-standalone installs

### ISSUE THREAD

**Issue by edmorley** [MayhemYDG/iltorb#61]
Hi!

If I run:
`docker run --rm -it node:9.7.1-slim yarn add brotli-size`
(which has a dependency on iltorb `^2.0.5`)

I get:
```
yarn add v1.5.1
info No lockfile found.
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
[1/1] ⠠ iltorb: installing standalone, skipping download.
[-/1] ⠠ waiting...
[-/1] ⠠ waiting...
[-/1] ⠠ waiting...
error An unexpected error occurred: "/node_modules/iltorb: Command failed.
Exit code: 1
Command: sh
Arguments: -c node ./scripts/install.js || node-gyp rebuild
Directory: /node_modules/iltorb
Output:
info install installing standalone, skipping download.
gyp info it worked if it ends with ok
gyp info using node-gyp@3.6.2
gyp info using node@9.7.1 | linux | x64
gyp ERR! configure error
gyp ERR! stack Error: Can't find Python executable \"python\", you can set the PYTHON env variable. gyp ERR! stack     at PythonFinder.failNoPython (/node_modules/node-gyp/lib/configure.js:483:19)
gyp ERR! stack     at PythonFinder.<anonymous> (/node_modules/node-gyp/lib/configure.js:397:16)
gyp ERR! stack     at F (/node_modules/which/which.js:68:16)
gyp ERR! stack     at E (/node_modules/which/which.js:80:29)
gyp ERR! stack     at /node_modules/which/which.js:89:16
gyp ERR! stack     at /node_modules/isexe/index.js:42:5
gyp ERR! stack     at /node_modules/isexe/mode.js:8:5
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:170:21)
gyp ERR! System Linux 4.4.111-boot2docker
gyp ERR! command \"/usr/local/bin/node\" \"/node_modules/iltorb/node_modules/.bin/node-gyp\" \"rebuild\"
gyp ERR! cwd /node_modules/iltorb
gyp ERR! node -v v9.7.1
gyp ERR! node-gyp -v v3.6.2
```

My understanding of the `"installing standalone, skipping download"` message is that it's for the case where one installs iltorb directly (eg `yarn add iltorb`) rather than the case where it's pulled in via a sub-dependency.

As such, it seems that in this case it's being shown incorrectly?

Many thanks :-)

**Comment by oohnoitz** [comment]
This should be addressed in the latest version `v2.0.9`.

**Comment by edmorley** [comment]
Works great now - thank you :-)
